opcua/common/node.py

```diff
@@ -519,11 +524,46 @@ def read_raw_history(self, starttime=None, endtime=None, numvalues=0):
         else:
             details.EndTime = ua.get_win_epoch()
         details.NumValuesPerNode = numvalues
-        details.ReturnBounds = True
+        details.ReturnBounds = returnBounds
         result = self.history_read(details)
         result.StatusCode.check()
         return result.HistoryData.DataValues
 
+    @staticmethod
+    def bulk_history_read(
+            client,  # UaClient or InternalServer
+            history_read_value_ids: list,  # List[HistoryReadValueId]
+            starttime: datetime = None,
+            endtime: datetime = None,
+            numvalues: int = 0
+    ) -> List[HistoryReadResult]:
+        """
+        Reads from multiple nodes and has returnBounds set to False.
+
+        Returns a list of the OPCUA-formatted results. For now, let the client handle the status codes how they want.
+        """
+        details = ua.ReadRawModifiedDetails()
+        details.IsReadModified = False
+        if starttime:
+            details.StartTime = starttime
+        else:
+            details.StartTime = ua.get_win_epoch()
+        if endtime:
+            details.EndTime = endtime
```

Comment:
So the default value for `endtime` is 1970-01-01T00:00:00Z?

Reply:
yep, that's correct. i was about to remove it but it looks like the opcua spec states that even if it wasn't specified, it would do the same thing. 

ill just add a comment here